content/applications/inventory_and_mrp/manufacturing/workflows/allocation.rst

```diff
@@ -0,0 +1,142 @@
+==================
+Allocation reports
+==================
+
+.. |SO| replace:: :abbr:`SO (sales order)`
+.. |SOs| replace:: :abbr:`SOs (sales orders)`
+.. |MO| replace:: :abbr:`MO (manufacturing order)`
+.. |MOs| replace:: :abbr:`MOs (manufacturing orders)`
+.. |RfQ| replace:: :abbr:`RfQ (request for quotation)`
+
+When fulfilling sales orders (SOs), or sourcing components for manufacturing orders (MOs), it is
+sometimes necessary to prioritize one |SO| or |MO| over another. In situations where there is
+insufficient stock on-hand to fulfill every |SO| or |MO|, ensuring that products and components are
+reserved for priority orders is essential.
+
+In Odoo *Manufacturing*, allocation reports are used on |MOs| to assign products to specific sales
+orders |SOs|, or components to specific |MOs|. This ensures the products or components are available
+for those orders, and are not used by mistake.
+
+Configuration
+=============
+
+To use allocation reports, the *Allocation Report for Manufacturing Orders* feature **must** be
+enabled. To do so, navigate to :menuselection:`Manufacturing app --> Configuration --> Settings`,
+and tick the checkbox next to :guilabel:`Allocation Report for Manufacturing Orders`. Then, click
+:guilabel:`Save`.
+
+.. image:: allocation/allocation-setting.png
+   :align: center
+   :alt: The Allocation Report for Manufacturing Orders setting.
+
+For products that are sold, it is also necessary to configure them so they can be included in |SOs|.
+To do so, begin by navigating to :menuselection:`Inventory --> Products --> Products`, and select a
+product. Under the :guilabel:`Product Name` field on the product form, make sure that the
+:guilabel:`Can Be Sold` checkbox is ticked.
+
+.. image:: allocation/can-be-sold.png
+   :align: center
+   :alt: The Can Be Sold checkbox on a product form.
+
+Allocate products
+=================
+
+To allocate products or components from an |MO| to an |SO|, or to a different |MO|, begin by
+navigating to :menuselection:`Manufacturing app --> Operations --> Manufacturing Orders`. Click
+:guilabel:`New` to create a new |MO|.
+
+On the |MO| form, select a product in the :guilabel:`Product` field, and specify the quantity to be
+produced in the :guilabel:`Quantity` field. Finally, click :guilabel:`Confirm` to confirm the |MO|.
+
+The rest of the allocation workflow depends on the current on-hand quantity of the product being
+manufactured, and whether or not there are any |SOs| or |MOs| which require the product, but have
+not already been allocated units.
+
+If there **ARE** existing |SOs| and |MOs| that require the product, **AND** there are too few units
```

Comment:
I understand this is important to distinguish, but I think the all-cap is a bit much. Suggesting to just leverage the bold formatting for emphasis :) 
```suggestion
If there **are** existing |SOs| and |MOs| that require the product, **and** there are too few units
```